presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/execution/thrift/ThriftSqlFunctionExecutor.java

```diff
@@ -119,6 +126,7 @@ private Block getResultBlock(ThriftUdfResult result, Type returnType)
             case PRESTO_THRIFT:
                 return getOnlyElement(page.getThriftPage().getThriftBlocks()).toBlock(returnType);
             case PRESTO_SERIALIZED:
+                checkState(blockEncodingSerde != null, "blockEncodingSerde not set");
```

Comment:
Those require non null is not necessary but they are used in a constructor next, which decides whether to requireNonNull on blockEncodingSerde itself (and usually does require).

Reply:
should I change it to checkArgument then?

Reply:
I think we can simply remove those 2 lines.